controllers/cartController.js

```diff
@@ -0,0 +1,21 @@
+const cartService = require("../services/cartService");
+const { asyncErrorHandler } = require("../middlewares/errorHandling");
+
+// 3. 장바구니 상품 삭제
+const deleteCart = asyncErrorHandler(async (req, res) => {
+  const { cartId } = req.params;
+
+  if (!cartId) {
+    const err = new Error("KEY_ERROR");
+    err.statusCode = 400;
+    throw err;
+  }
+  await cartService.deleteCart(cartId);
+  return res.status(200).json({
```

Comment:
delete의 경우 status code로 204를 많이 사용합니다.
```suggestion
  return res.status(204).json({
```

Reply:
status code 지적을 자주 받는데 그만큼 제가 확인하는 습관이 안 들어서 그런 것 같습니다 😂 status code 까지 꼼꼼히 고치겠습니다!